core/server/helpers/reading_time.js

```diff
@@ -48,11 +47,9 @@ module.exports = function reading_time(options) {// eslint-disable-line camelcas
         readingTimeSeconds += Math.max(i, 3);
     }
 
-    readingTimeMinutes = Math.round(readingTimeSeconds / 60);
+    readingTimeMinutes = Math.max(Math.round(readingTimeSeconds / 60), 1);
```

Comment:
Instead of this, couldn't we just use a `<=` comparison below? I think it would make readability a little easier

Reply:
Makes sense.